app/views/overview/_service-instance-row.html

```diff
@@ -0,0 +1,133 @@
+<div class="list-pf-item" ng-class="{ active: row.expanded }">
+  <div class="list-pf-container" ng-click="row.toggleExpand($event)">
+
+    <div class="list-pf-chevron">
+      <div ng-include src=" 'views/overview/_list-row-chevron.html' " class="list-pf-content"></div>
+    </div>
+
+    <div class="list-pf-content">
+      <div class="list-pf-name">
+        <h3>
+          <a ng-href="{{row.apiObject | navigateResourceURL}}">
+            <span ng-bind-html="row.displayName | highlightKeywords : row.state.filterKeywords"></span>
+          </a>
+          <div class="component-label" style="padding-top: 4px;">
+            {{row.apiObject.metadata.name}}
+          </div>
+        </h3>
+        <div class="status-icons">
+          <notification-icon ng-if="!row.expanded" alerts="row.notifications"></notification-icon>
+        </div>
+      </div>
+    </div>
+
+    <div
+      class="list-pf-content hidden-xs hidden-sm">
+      <div style="width: 300px">
+        <div>
+          <strong ng-if="!row.instanceBindings.length">No Bindings</strong>
+          <strong ng-if="row.instanceBindings.length">Bindings</strong>
+        </div>
+        <a g-if="row.instanceBindings.length >= 1" href="{{row.getSecretForBinding(row.instanceBindings[0]) | navigateResourceURL}}">{{row.instanceBindings[0].metadata.name}}</a><span ng-if="row.instanceBindings.length > 1">,&nbsp;</span>
```

Comment:
same point about other resources, we may want to just not make this a link right now